packages/frontend/app/templates/application.hbs

```diff
@@ -3,13 +3,13 @@
 <ApiVersionNotice />
 <UpdateNotification />
 <div
-  class="application-wrapper {{if this.currentUser.performsNonLearnerFunction 'show-navigation'}}"
+  class="application-wrapper{{if this.currentUser.performsNonLearnerFunction ' show-navigation'}}"
 >
   <IliosHeader />
   <div class="ilios-logo">
     <LinkTo @route="dashboard" title={{t "general.dashboard"}}>
       {{! template-lint-disable require-valid-alt-text }}
-      <img src="/assets/images/ilios-logo.svg" alt="" />
+      <img src="/assets/images/ilios-logo.svg" alt="" width="70" height="40" />
```

Comment:
I think a comment here on how these values were calculated or updating the commit message would be a good add. I was worried it would override what is an otherwise pretty nice auto scaling svg, but if I'm understanding it correctly this is really just the aspect ratio of the image?